e2e/test/iothub/messaging/MessageReceiveE2ETests.cs

```diff
@@ -31,94 +31,16 @@ public partial class MessageReceiveE2ETests : E2EMsTestBase
         private static readonly TimeSpan s_twentySeconds = TimeSpan.FromSeconds(20);
         private static readonly TimeSpan s_oneMinute = TimeSpan.FromMinutes(1);
 
-        [LoggedTestMethod]
-        public async Task Message_DeviceReceiveSingleMessage_Amqp()
```

Comment:
Removed web socket and X509 test combinations.

Reordered to have all tests of one type which vary by transport type together.